### PR TITLE
Change name of master branch to main.

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR converts all usage of the term 'master' (referring to the master branch) to 'main' (which will be the new name of that branch)
- Note that this PR should be approved before the branch name is changed to main, but this will be merged after the branch name change is pushed.

### CODE REVIEW

File: scripts/uberenv/uberenv.py [hunk 1]
```diff
@@ -319,7 +319,7 @@ def setup_paths_and_dirs(self):
     def clone_repo(self):
         if not os.path.isdir(self.dest_vcpkg):
             # compose clone command for the dest path, vcpkg url and branch
-            vcpkg_branch = self.project_opts.get("vcpkg_branch", "master")
+            vcpkg_branch = self.project_opts.get("vcpkg_branch", "main")
```
Comment (white238) [inline scripts/uberenv/uberenv.py:322]: vcpkg has not changed their branch from master.

File: src/conf.py [hunk 2]
```diff
@@ -81,8 +81,8 @@
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
-# The master toctree document.
-master_doc = 'index'
+# The main toctree document.
```
Comment (white238) [inline src/conf.py:84]: I suspect these were not supposed to be changed.

File: src/docs/doxygen/Doxyfile.in [hunk 3]
```diff
@@ -1318,7 +1318,7 @@ CHM_FILE               =
 HHC_LOCATION           =
 
 # The GENERATE_CHI flag controls if a separate .chi index file is generated
-# (YES) or that it should be included in the master .chm file (NO).
+# (YES) or that it should be included in the main .chm file (NO).
```
Comment (white238) [inline src/docs/doxygen/Doxyfile.in:1321]: Same here.

File: scripts/uberenv/uberenv.py (hunk 1 above)
Comment (rhornung67) [reply scripts/uberenv/uberenv.py:322]: Good catch. I'll fix that.

File: src/conf.py (hunk 2 above)
Comment (rhornung67) [reply src/conf.py:84]: I wasn't sure if those were our labels or used by the doc generation tool. I'll change back.

Comment (white238) [reply src/conf.py:84]: It's a sphinx option. not ours.

Comment (rhornung67) [reply src/conf.py:84]: Changed back.

File: scripts/uberenv/uberenv.py (hunk 1 above)
Comment (rhornung67) [reply scripts/uberenv/uberenv.py]: Changed back.

File: src/docs/doxygen/Doxyfile.in (hunk 3 above)
Comment (rhornung67) [reply src/docs/doxygen/Doxyfile.in]: Changed back.

File: scripts/gitsetup/setup-git-aliases.sh [hunk 4]
```diff
@@ -72,26 +72,26 @@ ${gconf} alias.undo "!bash -c \"git reset HEAD~ \""
 ## Usage: git squash-merge <branch-name>
 ##
 ## Merges the branch with the given name into the current branch,
-## typically, master using a squash-merge. All commits in the
+## typically main, using a squash-merge. All commits in the
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply scripts/gitsetup/setup-git-aliases.sh:75]: @kennyweiss I agree with that, but this PR was not intended to rework the docs, which need an overhaul. That would be a separate PR or set of PRs.

File: src/axom/mint/docs/sphinx/sections/citations.rst [hunk 5]
```diff
@@ -10,7 +10,7 @@
 .. _Axom Quick Start Guide: ../../../../../docs/sphinx/quickstart_guide/index.html
 .. _Conduit: https://llnl-conduit.readthedocs.io/en/latest/index.html
 .. _Blueprint: https://llnl-conduit.readthedocs.io/en/latest/blueprint.html
-.. _RAJA: https://raja.readthedocs.io/en/master/index.html
+.. _RAJA: https://raja.readthedocs.io/en/main/index.html
```
[the remark this answers is not in the document]
Comment (rhornung67) [reply src/axom/mint/docs/sphinx/sections/citations.rst:13]: For RAJA, main is always the same as latest. RTD generates latest from the latest tag and the RAJA main branch only contains merge commits before releases, which are then tagged.

**Pull request merged**